README.md

```diff
@@ -58,55 +58,152 @@ documentation](https://www.packer.io/docs).
+7. Paste in the **big number** *prefixed by the letter 'c'*.  The *"c*" indicates
+   the images are *cache images*.  For example, if the url was `http://.../12345`
+   you would paste in `c12345` as the value for `_BUILT_IMAGE_SUFFIX:`.
+
+8. Open up a PR with this change, and push it.  Once all tests pass and you're
    satisfied with the image changes, ask somebody to review/approve both
    PRs for merging.  If you're feeling generous, perhaps provide cross-links
    between the two PRs in comments, for future reference.
 
-8. After all the PRs are merged, you're done.  You may now attend to the little
-   dog begging you for a walk for the last hour.  Hurry!  Little-dogs, do not
-   have big-dog bladders!
+9. After all the PRs are merged, you're done.
+
+
+
+
+## The image-builder image (overview step 1)
+
+Google compute engine (GCE) does not provide a wide selection of ready-made
+VM images for use.  Instead, a lengthy and sophisticated process is involved
+to prepare, properly format, and import external VM images for use.  In order
+to perform these steps within automation, a dedicated VM image is needed which
+itself has been prepared with the necessary incantations, packages, configuration,
+and magic license keys.
+
+For normal day-to-day use, this process should not need to be modified or
+maintained much.  However, on the off-chance that's ever not true, here is
+an overview of the process followed **by automation** to produce the
+*image-building VM image*:
+
+1. Build the container defined in the `ci` subdirectory's `ContainerFile`.
+   Start this container with a copy of the current repository code provided
+   in the `$CIRRUS_WORKING_DIR` directory. For example on a Fedora host:
+
+   ```
+   podman run -it --rm --security-opt label=disable -v $PWD:$PWD -w $PWD ci_image
+   ```
+
+2. From within the `ci` container (above), in the repository root volume, execute
+   the  `make image_builder` target.
+
+3. The `image_builder/gce.yml` file is converted into JSON format for
+   consumption by the [Hashicorp *packer* utility](https://www.packer.io/).
+   This generated file may be ignored, *make* will be regenerate it upon
+   any changes to the YAML file.
+
+4. Packer will spin up a GCE VM based on CentOS, installs necessary packages
+   and attaches a [nested-virtualization "license" to the
+   VM](https://cloud.google.com/compute/docs/instances/enable-nested-virtualization-vm-instances#enablenestedvirt).  Be patient until this process completes.
 
+5. Near the end of the process, the GCE VM delete temporary files, ssh host keys,
+   etc. and then shut down.
 
-## Container images
+6. Packer should then automatically call into the google cloud APIs and
+   coordinate conversion of the VM disk into a bootable image.  Again,
+   please be patient for this process to complete, it may take several minutes.
+
+7. When finished, packer will writes the freshly created image name and other
```

Comment:
All these changes! Should be "will **write**", not writes

Reply:
they're not changes, they're FIXES :grin: